server/routes/addPatient/hospitalSelect.ts

```diff
@@ -0,0 +1,14 @@
+import HospitalSelectRoutes from '../selectHospital/hospitalSelect'
+import MovePrisonerService from '../../services/movePrisonerService'
+import PrisonerSearchService from '../../services/prisonerSearchService'
+
+export default class AddPatientHospitalSelectRoutes extends HospitalSelectRoutes {
+  constructor(movePrisonerService: MovePrisonerService, prisonerSearchService: PrisonerSearchService) {
```

Comment:
bit naff that the add patient route calls the move prisoner service (since that is what the select hospital calls), will fix in next PR